index.html

```diff
@@ -1544,7 +1544,7 @@ <h5 style="margin:auto 4px">${row.team.tla}</h5>
 
     function getMatches()
     {
-        const url = `${baseUrl}/competitions/2000/matches`
+        const url = `https://cors-anywhere.herokuapp.com/${baseUrl}/competitions/2000/matches`
```

Comment:
This demo of CORS Anywhere should only be used for development purposes

Reply:
same!?